riotbuild/Dockerfile

```diff
@@ -13,22 +13,19 @@
 #      # docker pull riot/riotbuild
 # 
 # Use own build image:
-# 1. Build own image based on latest base OS image:
+# 1. Build own image based on latest base OS image (from the riotbuild directory):
 #      # docker build --pull -t riotbuild .
 #
 # Usage:
 # 1. cd to riot root
 # 2. # docker run -i -t -u $UID -v $(pwd):/data/riotbuild riotbuild ./dist/tools/compile_test/compile_test.py
 
-FROM ubuntu:bionic
+FROM riot/static-test-tools:latest
```

Comment:
We could also directly use `riot/riotdocker-base` here. I don't have a strong opinion here, the current approach results in the same `riot/riotbuild` image. Thoughts ?

Reply:
I think it needs to be this way for a while.